packages/rolldown/src/cli/commands/help.ts

```diff
@@ -0,0 +1,39 @@
+import { logger } from '../utils'
+import {
+  version,
+  description,
+} from '../../../package.json' assert { type: 'json' }
+import { CLI_OPTIONS } from '../options'
+import { bold, cyan, underline } from '../colors'
+
+const HELP_TEMPLATE = `rolldown version ${version}
+
+${description}
```

Comment:
> Fast JavaScript/TypeScript bundler in Rust with Rollup-compatible API. (rolldown v0.12.2)                                                                                        
Can you keep the layout as before?
![Uploading image.png…]()

Reply:
current output and before:

<img width="1448" alt="image" src="https://github.com/user-attachments/assets/56eab161-a830-4970-8366-3d713d36bcf3">